<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>481</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDPoliklinikLabel">
      <property name="text">
       <string>ID Poliklinik</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="EditPoli"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaPoliklinikLabel">
      <property name="text">
       <string>Nama Poliklinik</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cmbPoli">
      <item>
       <property name="text">
        <string>Pelayanan konsultasi dan administrasi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pelayanan kesehatan dasar umum</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pelayanan kesehatan gigi dan mulut</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pelayanan kesehatan anak (pediatri)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pelayanan obstetri dan ginekologi</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="keteranganLabel">
      <property name="text">
       <string>Keterangan</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EditKeterangan"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>491</width>
     <height>192</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>163</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID Poliklinik</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama Poliklinik</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Keterangan</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineCari">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>310</y>
     <width>491</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCetak">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cetak</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
